errors/errors.go

```diff
@@ -61,7 +61,7 @@ func (e Error) Error() string {
 func CreateInternalServerError(message string) Error {
 	return Error{
 		Status:  http.StatusInternalServerError,
-		Message: GetEmitter() + "." + message,
+		Message: concat(GetEmitter(), message),
```

Comment:
Not a big fan of that. I have the feeling that we change easy code to complex code for no reason.